roles/ceph-config/templates/ceph.conf.j2

```diff
@@ -73,6 +73,12 @@ admin socket = {{ rbd_client_admin_socket_path }}/$cluster-$type.$id.$pid.$cctid
 log file = {{ rbd_client_log_file }} # must be writable by QEMU and allowed by SELinux or AppArmor
 {% endif %}
 
+{% if openstack_config | bool %}
+[client.manila]
+client mount gid = 0
+client mount uid = 0
+{% endif %}
```

Comment:
This should be set only for clients node AND when client.manila is present in the `openstack_keys` list (ie: not hardcoding the manila keyring name)

Reply:
Yeah, indeed. Actually probably better idea would be to include clients conditionally or allow users to override their names but I tried to follow current concept...